roles/cert_manager/tasks/cleanup.yml

```diff
@@ -14,6 +14,33 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Delete deployment of cert-manager
+  kubernetes.core.k8s:
+    state: absent
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit)}}"
+    kind: Deployment
+    NameSpace: cert-manager
+    label_selectors:
+      - app.kubernetes.io/instance=cert-manager
+
+- name: Delete CRDs related to cert-manager
+  kubernetes.core.k8s:
+    state: absent
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit)}}"
+    kind: CustomResourceDefinition
+    NameSpace: "{{ cifmw_cert_manager_operator_namespace }}"
+    name: "{{ item }}"
+  loop:
```

Comment:
is that loop static and always the same? Should we consider having that as a parameter to the role?

Reply:
Nope, Replaced it with a var